src/initializers/staticFile.ts

```diff
@@ -231,3 +234,25 @@ export class StaticFile extends Initializer {
     );
   }
 }
+
+async function asyncStats(file: string): Promise<{ [key: string]: any }> {
```

Comment:
```suggestion
async function asyncStats(file: string): Promise<ReturnType<typeof fs.statSync>> {
```
Doing this will fix some comments I made elsewhere.